src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 40;
+  let day = 1;
+
+  for (; day < days; day++) {
+    cost += 40;
+  }
+
+  if (day >= 3 && day <= 6) {
+    return (cost - 20);
+  } else if (day >= 7) {
+    return (cost - 50);
+  } else {
+    return cost;
+  }
```

Comment:
Also do not use else statement
![2023-02-10_15h35_46](https://user-images.githubusercontent.com/61904995/218105199-7158192c-6d25-4ef8-a8f6-b9d75c953aa4.png)